src/pages/settings/SettingFormInput.tsx

```diff
@@ -61,15 +63,27 @@ const SettingFormInput: FC<Props> = ({
       <Button appearance="positive" type="submit">
         Save
       </Button>
-      {canBeReset && (
+      {configField.isUserDefined && (
         <Button
-          className="reset-button"
+          appearance="base"
+          className="reset-or-delete-button"
+          onClick={onDelete}
+        >
+          Delete key
```

Comment:
```suggestion
          Delete user setting
```
Suggestion for a bit better copy. I am open to further suggestions, as I am not 100% sure this is the right way to label it to make it precisely understood.